### DISCUSSION TITLE
Tap to dismiss alarm

### DISCUSSION THREAD

**Discussion by lindely** [BlackyHawky/Clock#54] [Ideas]
Hi!

I don't know if any requests are accepted, but I will give it a shot anyway. I currently use an old alarm clock app simply because it is the only one I can find that allows me to tap a "dismiss alarm" button when the alarm goes off. When I wake up in the morning, the last thing I want is to perform a swipe and fail multiple times in a row (plus I dislike swiping, and that's a me thing). However, that alarm app has no stopwatch or timer feature, so I use a different clock app for that.

So, that's a very roundabout way of asking: would the developers of this app be interested in adding an option to allow for tapping to dismiss an alarm instead of swiping?

Thank you for reading :-)

**Comment by BlackyHawky** [comment]
@lindely 
Hi,

Of course requests are accepted even if, as indicated in the [Readme](https://github.com/BlackyHawky/Clock?tab=readme-ov-file#common-issues), I wouldn't necessarily be able to solve them all.

In any case, your request is a good idea, so I've transferred it to the new issue #58.

You'll be happy to know that your feature has been resolved (commit 93d9c7b) and will be available in the next version. 😉

**Reply by lindely** [reply]
Fantastic, thank you so much :-D
